lib/locales/es.yml

```diff
@@ -1,27 +1,92 @@
+  expectation_HasPatternMatching: '%{binding} %{must} utilizar pattern matching'
+  expectation_HasPredicate: 'el predicado %{binding} %{must} estar declarado'
+  expectation_HasProcedure: 'el procedimiento %{binding} %{must} estar declarado'
+  expectation_HasProcedure_named: '%{binding} %{must} declarar un procedimiento %{target}'
+  expectation_HasProcedure_like: '%{binding} %{must} declarar un procedimiento parecido a %{target}'
+  expectation_HasProcedure_tail: '%{binding} %{must} declarar procedimientos'
   expectation_HasRedundantBooleanComparison: '%{binding} hace comparaciones booleanas innecesarias'
   expectation_HasRedundantGuards: '%{binding} tiene guardas innecesarias'
   expectation_HasRedundantIf: '%{binding} tiene ifs innecesarios'
   expectation_HasRedundantLambda: '%{binding} tiene lambdas innecesarias'
   expectation_HasRedundantParameter: '%{binding} tiene parámetros innecesarios (se pueden eliminar mediante point-free)'
   expectation_HasRepeat: '%{binding} %{must} usar una repetición simple (sentencia <i>repeat</i>)'
   expectation_HasRepeatOf: '%{binding} %{must} usar una repetición simple (sentencia <i>repeat</i>) de %{target}'
-  expectation_HasTypeDeclaration: '%{must} existir una declaración para el sinónimo de tipo %{binding}'
-  expectation_HasTypeSignature: La firma para %{binding} %{must} ser declarada
-  expectation_HasUsage: '%{binding} %{must} utilizar %{target}'
-  expectation_HasVariable: '%{binding} %{must} utilizar variables'
+  expectation_HasRule: 'la regla %{binding} %{must} estar declarada'
+  expectation_HasRule_named: '%{binding} %{must} declarar una regla %{target}'
+  expectation_HasRule_like: '%{binding} %{must} declarar una regla parecida a %{target}'
+  expectation_HasRule_tail: '%{binding} %{must} declarar reglas'
+  expectation_HasSwitch: '%{binding} %{must} utilizar un switch'
+  expectation_HasTypeAlias: 'el sinónimo de tipo %{binding} %{must} estar declarado'
+  expectation_HasTypeAlias_named: '%{binding} %{must} declarar un sinónimo de tipo %{target}'
+  expectation_HasTypeAlias_like: '%{binding} %{must} declarar un sinónimo de tipo parecido a %{target}'
+  expectation_HasTypeAlias_tail: '%{binding} %{must} declarar sinónimos de tipo'
+  expectation_HasTypeSignature: La firma %{binding} %{must} ser declarada
+  expectation_HasTypeSignature_named: '%{binding} %{must} declarar una firma %{target}'
+  expectation_HasTypeSignature_like: '%{binding} %{must} declarar una firma para algo parecido %{target}'
+  expectation_HasTypeSignature_tail: '%{binding} %{must} declarar firmas'
+  expectation_HasUnifyOperator: '%{binding} %{must} utilizar el operador de unificación (=)'
+  expectation_HasUsage: '%{binding} %{must} ser utilizado'
+  expectation_HasUsage_named: '%{binding} %{must} utilizar %{target}'
+  expectation_HasUsage_like: '%{binding} %{must} delegar en algo parecido a %{target}'
+  expectation_HasUsage_tail: '%{binding} %{must} delegar'
```

Comment:
This is a good sample of the new power of Mulang: we can check if the student has divided her problem into subproblems, without forcing a name, or we can check that she has written at least a type signature @faloi @asanzo